package.json

```diff
@@ -16,10 +16,13 @@
     "axios": "^1.5.1",
     "gh-pages": "^5.0.0",
     "hamburger-react": "^2.5.0",
+    "isemail": "^3.2.0",
```

Comment:
Why you have installed `isemail` package? What is use of this pakage in this project?